source/github-events/on-pr-merge-panel-open.ts

```diff
@@ -10,7 +10,10 @@ const delegateHandler = mem((callback: EventListener) => (event: delegate.Event)
 
 const sessionResumeHandler = mem((callback: EventListener) => async (event: Event) => {
 	await Promise.resolve(); // The `session:resume` event fires a bit too early
-	callback(event);
+	// Avoid triggering the callback when a page with a non-empty comment field is reloaded #3932
+	if ((event as CustomEvent).detail.targetId !== 'new_comment_field') {
```

Comment:
Not sure about the type coercion. Maybe the callback should be defined as taking a `CustomEvent` in the first place?

Reply:
Yes if it works

Reply:
It can work without any assertions if we declare `'session:resume'` as a `CustomElement`:
```typescript
declare global {
	interface DocumentEventMap {
		'session:resume': CustomEvent
	}
}
```